laura/portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,14 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  
-    <ul>
-        <li><a href="index.html" class="navbar">Home</a>
-            <a href="mood.html" class="navbar">Mood generator</a></li>
-    </ul>
-
   <body>
+    <nav>
+        <ul>
+            <li><a href="index.html" class="navbar">Home</a>
+                <a href="mood.html" class="navbar">Mood generator</a></li>
+        </ul>
```

Comment:
It's better to have each navigation element inside it's own li.
Simple example could be found here: https://www.w3schools.com/css/css_navbar.asp